src/super_gradients/training/models/detection_models/yolov5.py

```diff
@@ -394,6 +402,20 @@ def prep_model_for_conversion(self, input_size: Union[tuple, list] = None, **kwa
     def get_include_attributes(self) -> list:
         return ["grid", "anchors", "anchors_grid"]
 
+    def replace_head(self, new_num_classes=None, new_head=None):
+        if new_num_classes is None and new_head is None:
+            raise ValueError("At least one of new_num_classes, new_head must be given to replace output layer.")
+        if new_head is not None:
+            self._head = new_head
+        else:
+            self.arch_params.num_classes = new_num_classes
+            new_last_layer = Detect(new_num_classes, self._head.anchors, channels=[self._head.width_mult(v) for v in (256, 512, 1024)])
```

Comment:
@shaydeci - whay are these magic numbers?
(256, 512, 1024)???

Reply:
Output channels of previous modules in YoLoV5Head, this is exactly how it's also initialised in its constructor as well.

Reply:
true. but perhaps we should consider moving this into the params dictionary in the future